docs: add known issue PE-7313 (#7798)
spectrocloud/librarium: docs/docs-content/troubleshooting/edge/edge.md

The following are common scenarios that you may encounter when using Edge.

> ## Scenario - Content-Length: 0 Errors during Content Synchronization

Reviewer: ```suggestion
## Scenario - `content-length: 0` Errors during Content Synchronization
```

Unintended or non-graceful reboots during content bundle push operations can cause inconsistency in the primary

> registry, resulting in 0-byte blob files and subsequent Content-Length: 0 errors during content synchronization.

Reviewer: ```suggestion
registry, resulting in 0-byte blob files and subsequent `content-length: 0` errors during content synchronization.
```

When operating a locally managed cluster, if you observe the `content-length: 0` error during content sync, follow the

> steps to below to verify the error was caused by inconsistency in the primary registry and fix the issue.

Reviewer: ```suggestion
steps below to verify whether the error was caused by an inconsistency in the primary registry and to fix the issue.
```

### Debug Steps

> 1. Issue the following command and observe if there are files that are 0 byte in size.

Reviewer: ```suggestion
1. Issue the following command and observe if there are files that are zero bytes in size.
```

   ```bash
   ls -la /usr/local/spectrocloud/peerbundle/sha256/
   ```

>    If you observe output similar to the following, where the size of the file is zero, proceed to the next step.

Reviewer: ```suggestion
   If you observe output similar to the following, where the file size is zero, proceed to the next step.
```
